package.json

```diff
@@ -1604,7 +1604,7 @@
                             "type": "string"
                         }
                     ],
-                    "default": "docker build --pull --rm -f \"${dockerfile}\" -t ${tag} \"${context}\"",
+                    "default": "${config:docker.dockerPath} build --pull --rm -f \"${dockerfile}\" -t ${tag} \"${context}\"",
```

Comment:
Rather than always running `docker ...`, we can pick up the setting they apply with `docker.dockerPath`. In contrast to `docker.composeCommand`, that setting is _not_ unset by default, because we aren't attempting to do any autodetection.